.github/workflows/main.yml

```diff
@@ -21,11 +21,11 @@ jobs:
     if: ${{ success() || ( needs.ci-approval.result == 'skipped' && ( github.event_name == 'push' || github.actor == 'dependabot[bot]' ) ) }}
     runs-on: ubuntu-latest
     steps:
-    -   uses: actions/checkout@v3
+    -   uses: actions/checkout@v4.1.7
```

Comment:
Is it possible to just use v4 instead of the full version?

Reply:
Updated. Thanks.